apps/condo/domains/notification/adapters/firebaseAdapter.js

```diff
@@ -178,7 +180,7 @@ class FirebaseAdapter {
                     ...extraPayload,
                 }
 
-            target.push(pushData)
+            if (!APPS_WITH_DISABLED_NOTIFICATIONS.includes(data.app)) target.push(pushData)
```

Comment:
So, if there are no configuration, then:

APPS_WITH_DISABLED_NOTIFICATIONS will be null

and this command will be null.includes(data.app) which will be unhandled exception

Reply:
!

Reply:
Why have you implemented this stuff using two different instructions? 

In Apple adapter you use continue, here you use includes 

+ What about Huawei adapter?

Reply:
These are two different adapters, they have the same essence but are implemented differently in some logic. This task is not about refactoring and bringing all adapters to a single image. This is a task about testing a hypothesis, which should fix a certain error.

Reply:
What about huawei? 

I do not see a blocker in writing code in same way